src/master-playlist-controller.js

```diff
@@ -330,6 +330,53 @@ export class MasterPlaylistController extends videojs.EventTarget {
     this.abrTimer_ = null;
   }
 
+  getAudioTrackPlaylists_() {
+    const playlists = [];
+    const master = this.master();
+
+    if (!master && !master.mediaGroups && !master.mediaGroups.AUDIO) {
+      return playlists;
```

Comment:
When first reading through this function I didn't get why all playlists are returned if there's no audio groups, then later realized it's for audio only playlists. Should we add a check for isAudioOnly as part of this conditional or add a comment to the method to note that this assumes either an active audio group or an audio only playlist?